src/group_theory/order_of_element.lean

```diff
@@ -774,6 +774,28 @@ end
 
 attribute [to_additive card_nsmul_eq_zero] pow_card_eq_one
 
+/-- If `gcd(|G|,k)=1` then the `k`th power map is a bijection -/
+def pow_coprime (h : nat.coprime (fintype.card G) n) : G ≃ G :=
```

Comment:
Can you either use `n` in the docstring or `k` in the lemma, so that they match?